youtube_dl/extractor/megacartoons.py

```diff
@@ -0,0 +1,53 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+
+from .common import InfoExtractor
+
+
+class MegaCartoonsIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?megacartoons\.net/(?P<id>[a-z-]+)/'
+    _TEST = {
+        'url': 'https://www.megacartoons.net/help-wanted/',
+        'md5': '4ba9be574f9a17abe0c074e2f955fded',
+        'info_dict': {
+            'id': 'help-wanted',
+            'title': 'Help Wanted',
+            'ext': 'mp4',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'description': 'Help Wanted: Encouraged by his best friend, Patrick Starfish, '
+                           'SpongeBob overcomes his fears and finally applies for that '
+                           'dream job as a fry cook at the Krusty Krab. Challenged by the '
+                           'owner, Mr. Krabs, and his assistant Squidward, to prove himself '
+                           'worthy of the job, SpongeBob rises to the occasion, with the help '
+                           'of one very special spatula, by feeding a sea of ravenous anchovies.'
+        }
+    }
+
+    def _real_extract(self, url):
+        # ID is equal to the episode name
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+
```

Comment:
Consider using the `_search_json_ld()` method:
```suggestion

       info = self._search_json_ld(webpage, video_id, default={})
```
Then the fields from this JSON 
```json
    {
      '@type': 'VideoObject',
      '@id': 'https://www.megacartoons.net/help-wanted/#video',
      'name': 'Help Wanted - SpongeBob SquarePants',
      'isPartOf': {
        '@id': 'https://www.megacartoons.net/help-wanted/#article'
      },
      'thumbnailUrl': 'https://www.megacartoons.net/wp-content/uploads/Help-Wanted.jpg',
      'description': 'Help Wanted: Encouraged by his best friend, Patrick Starfish, SpongeBob overcomes his fears and finally applies for that dream job as a fry cook at the Krusty Krab. Challenged by the owner, Mr. Krabs, and his assistant Squidward, to prove himself worthy of the job, SpongeBob rises to the occasion, w',
      'uploadDate': '2020-02-23',
      'contentUrl': 'https://ww.megacartoons.net/video/SpongeBob-SquarePants-Help-Wanted.mp4',
      'isFamilyFriendly': 'true',
      'inLanguage': 'en-US'
    }
```
should be set in `info`: `name` -> `title`, `thumbnailUrl` -> `thumbnail`, `uploadDate` -> `upload_date`, `contentUrl` -> `url`. However `description` gets clipped in the test video.

In the test video this bypasses the need to search for the video URL elsewhere in the page.